test/rest-api/response-patching.test.ts

```diff
@@ -23,6 +23,14 @@ describe('REST: Response patching', () => {
           app.post('/posts', (req, res) => {
             res.status(200).json({ id: 101 }).end()
           })
+
+          app.get('/posts', (req, res) => {
+            res.status(200).json({ id: 101 }).end()
+          })
+
+          app.head('/headtest', (req, res) => {
+            res.status(200).end()
+          })
```

Comment:
```suggestion
```

Reply:
Thank you so much for your support. Updated the test according to your changes :)